code/modules/mob/living/carbon/xenomorph/evolution.dm

```diff
@@ -370,7 +370,8 @@
 	while(new_xeno.upgrade_possible() && new_xeno.upgrade_stored >= new_xeno.xeno_caste.upgrade_threshold)
 		new_xeno.upgrade_xeno(new_xeno.upgrade_next(), TRUE)
 	var/obj/screen/zone_sel/selector = new_xeno.hud_used.zone_sel
-	selector.set_selected_zone(zone_selected, new_xeno)
+	if(selector)
+		selector.set_selected_zone(zone_selected, new_xeno)
```

Comment:
```suggestion
	selector?.set_selected_zone(zone_selected, new_xeno)
```

Reply:
harder to read imo, but smaller and functionally the same, its in